profiler/src/ProfilerEngine/Datadog.Profiler.Native/Configuration.cpp

```diff
@@ -604,3 +615,30 @@ bool Configuration::IsEnvironmentValueSet(shared::WSTRING const& name, T& value)
     value = result;
     return true;
 }
+
+void Configuration::ExtractSsiState(bool& ssiDeployed, bool& ssiEnabled)
+{
+    // if the profiler has been deployed via Single Step Instrumentation,
+    // the DD_INJECTION_ENABLED env var exists.
+    // if the profiler has been activated via Single Step Instrumentation,
+    // the DD_INJECTION_ENABLED env var should contain "profiling" (it is a list of SSI installed products)
+    //
+    if (!shared::EnvironmentExist(EnvironmentVariables::SsiDeployed))
+    {
+        ssiDeployed = false;
+        ssiEnabled = false;
+        return;
+    }
+
+    ssiDeployed = true;
+
+    auto r = shared::Trim(shared::GetEnvironmentValue(EnvironmentVariables::SsiDeployed));
+    if (r.empty())
+    {
+        ssiEnabled = false;
+        return;
+    }
+
+    auto pos = r.find(WStr("profiling"),0);
```

Comment:
no need to specify the start position `(0)` . It's the default.

Reply:
✅ done